doc/NPCs.md

```diff
@@ -185,7 +185,8 @@ The dynamic line is chosen based on whether the NPC has a specific trait.  Both
 ```
 
 ### Based on trait or mutation flag possessed by the player character
-The dynamic line is chosen based on whether the player character has a trait with a specific flag.  Both entries are optional. The `u_has_trait_flag` string should be a valid trait flag.  The line from `yes` will be shown if the character has any traits with the flag, otherwise the line from `no`.
+The dynamic line is chosen based on whether the player character has a trait with a specific flag.  Both entries are optional. The `u_has_trait_flag` string should be a valid trait flag.  The line from `yes` will be shown if the character has any traits with the flag, otherwise the line from `no`.   The special trait flag "MUTATION_THRESHOLD" checks to see if the player has crossed a mutation threshold.
```

Comment:
```suggestion
The dynamic line is chosen based on whether the player character has a trait with a specific flag.  Both entries are optional. The `u_has_trait_flag` string should be a valid trait flag.  The line from `yes` will be shown if the character has any traits with the flag, otherwise the line from `no`.  The special trait flag "MUTATION_THRESHOLD" checks to see if the player has crossed a mutation threshold.
```